core/src/test/java/com/linecorp/armeria/internal/metric/RequestMetricSupportTest.java

```diff
@@ -60,31 +60,50 @@ public void http() {
         ctx.logBuilder().requestLength(123);
 
         Map<String, Double> measurements = measureAll(registry);
-        assertThat(measurements).containsEntry("foo.activeRequests#value{method=POST}", 1.0)
-                                .containsEntry("foo.requestDuration#count{method=POST}", 0.0)
-                                .containsEntry("foo.requestDuration#totalTime{method=POST}", 0.0)
-                                .containsEntry("foo.requestLength#count{method=POST}", 0.0)
-                                .containsEntry("foo.requestLength#total{method=POST}", 0.0);
+        assertThat(measurements).containsEntry("foo.activeRequests#value{method=POST}", 1.0);
+
+        ctx.logBuilder().responseHeaders(HttpHeaders.of(200));
+        ctx.logBuilder().responseLength(456);
 
         ctx.logBuilder().endRequest();
+        ctx.logBuilder().endResponse();
+
         measurements = measureAll(registry);
-        assertThat(measurements).containsEntry("foo.activeRequests#value{method=POST}", 1.0)
-                                .containsEntry("foo.requestDuration#count{method=POST}", 1.0)
-                                .containsEntry("foo.requestLength#count{method=POST}", 1.0)
-                                .containsEntry("foo.requestLength#total{method=POST}", 123.0);
+        assertThat(measurements).containsEntry("foo.activeRequests#value{method=POST}", 0.0)
+                                .containsEntry("foo.requests#count{method=POST,result=success,status=200}", 1.0)
```

Comment:
find that `request` metrics which contain status code is a bit akward, but to split it, we will need to split `RequestMetrics` and `ResponseMetrics` which @trustin not really interested it. Could you give me your comment?

Reply:
I think it's not awkward at all because because a user would want to know the stats like 'the request length of the requests that ende up with non-200 statuses'. i.e. no need to split at all in my opinion.